src/components/ChatRoomPage/ChatInput.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { currentUserState, allChatRoomsInfoState } from '../../state/atom';
+import styled from 'styled-components';
+import { IoIosSend } from 'react-icons/io';
+import { IChat, IUser } from '../../interface/interface';
+
+type ChatInputProps = {
+  friendInfo: IUser;
+  chatList: IChat[];
+};
+
+const ChatInput = ({ friendInfo, chatList }: ChatInputProps) => {
+  const currentUser = useRecoilValue(currentUserState);
+  const [text, setText] = useState('');
+  const [newChatList, setNewChatList] = useState(chatList);
+  const [allChatRoomsInfo, setAllChatRoomsInfo] = useRecoilState(
+    allChatRoomsInfoState
+  );
+
+  const addChat = () => {
+    setNewChatList([
+      ...newChatList,
+      { userId: currentUser.userId, content: text, time: currentTime() },
+    ]);
+  };
+
+  const currentTime = () => {
+    let hours = ('0' + new Date().getHours()).slice(-2);
+    let minutes = ('0' + new Date().getMinutes()).slice(-2);
+
+    return hours + ':' + minutes;
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!text.trim().length) {
+      return;
+    }
+
+    addChat();
+
+    setText('');
```

Comment:
```suggestion
    if (text.trim()) {
      setNewChatList([
        ...newChatList,
        { userId: currentUser.userId, content: text, time: currentTime() },
      ]);
      setText('');
    }
```

addChat가 여기서만 쓰이는 함수이고 비교적 간단한 로직이니, 내부에서 작성해줘도 될 것 같아요!
추가로, trim()을 했을 때 남아있는 문자열의 여부 자체만으로 플래그처리를 할 수 있을 것 같아요!!